{{cookiecutter.repo_name}}/Pipfile

```diff
@@ -3,21 +3,21 @@ url = "https://pypi.python.org/simple"
 verify_ssl = true
 
 [packages]
-django = "<2.2"
+django = "<=3.1"
```

Comment:
shouldn't this be something more like `~3.1` or `<3.2`? This prevents those micro fixes from getting pulled in.